internal/pkg/logging/logging.go

```diff
@@ -50,7 +50,7 @@ func (l *LogEntry) Warn(args ...interface{}) {
 
 // Error wraps the logrus Error function
 func (l *LogEntry) Error(err interface{}, args ...interface{}) {
```

Comment:
Maybe we should make this an `error` instead of removing the type safety from `WithError`?

Reply:
Or is this really meant to be a `string`?

Reply:
We have a lot of places where we're passing in a string with `logger.Error(...)`, I can change those if we want to keep the type safety though.

Reply:
I'm gonna do this in a separate followup PR